jparest/src/main/java/uk/gov/homeoffice/digital/sas/jparest/utils/EntityValidator.java

```diff
@@ -13,12 +14,13 @@
 
 import static java.util.stream.Collectors.groupingBy;
 
-public class ValidatorUtils {
+@Component
+public class EntityValidator {
```

Comment:
Now that you renamed this to EntityValidator, maybe the package should also be renamed to: `uk.gov.homeoffice.digital.sas.jparest.validation` ?

Reply:
good spot, have renamed.